.prettierignore

```diff
@@ -12,9 +12,12 @@ LICENSE
 
 ## same as in .gitignore
 # dependencies
-node_modules
-.pnp
-.pnp.js
+.yarn/*
+!.yarn/releases
+!.yarn/plugins
```

Comment:
```suggestion
```

You probably don't want prettier to touch these files

Reply:
Yeah you are right! Same for ESLint and Markdownlint. What is interesting though is that I just ran `yarn fix:prettier` and did not notice any `.yarn/*` paths in stdout. I guess that something in [my approach to config files](https://github.com/prettier/prettier/issues/6280#issuecomment-509568953) ends up excluding all folders starting with the dot by default. For that reason, I had to add `!/.github/` a while ago (see above) — otherwise the yamls were not formatted.

I’ll see what's best: to keep things as is or to ‘break the symmetry’ between the ignore files. Gonna switch to work work till the next weekend 🙂